modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/test/testJS/select_field.js

```diff
@@ -74,14 +74,38 @@ describe(
 		);
 
 		describe(
+			'.clickSelectTrigger()',
+			function() {
+				it(
+					'should has foucus in select aftet opened',
```

Comment:
fix sentence:

should activate the select after clicking
should add the 'active' CSS class on the select after clicking
**should add the focus class when opened** (this one matches better the other test for the closing case)

right now 'should has' is not syntactically correct.